provision.sh

```diff
@@ -1,5 +1,5 @@
-#!/bin/sh
 #!/bin/bash
+#!/bin/sh
```

Comment:
https://stackoverflow.com/a/42966709 states that there can only be one shebang, so it will attempt to launch bash, and if it is not available, report it cannot find bash. If you want it to attempt bash, then default to sh, you need to create a minor program to send the file to the correct shell, and make that your shebang. it would look like this:
```
#!/bin/sh
if which /bin/bash
then
/bin/bash $1
else
/bin/sh $1
fi

Reply:
That code doesn't work, but you're correct that there can only be one shebang, so this needs to change.

We enforce the bash for all our scripts in 
  https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/script
for about 7 years now and we haven't received any complaints about that.
So, let's just be consistent with those scripts and only use "bash". (And if someone submits a PR for a good working "if" approach, we'll apply it there too).